src/app/content/components/Sidebar.tsx

```diff
@@ -219,43 +362,84 @@ export class Sidebar extends Component<SidebarProps> {
 
   public componentDidUpdate() {
     this.scrollToSelectedPage();
+    this.expandCurrentChapter();
   }
 
   private scrollToSelectedPage() {
-    scrollTocSectionIntoView(this.sidebar, this.activeSection);
+    scrollTocSectionIntoView(this.sidebar.current, this.activeSection.current);
   }
 
-  private renderTocNode = (book: Book, {contents}: ArchiveTree) => <nav>
-    <ol>
-      {contents.map((item) => {
-        const active = (!!this.props.page) && stripIdVersion(item.id) === this.props.page.id;
+  private expandCurrentChapter() {
```

Comment:
this function is a little complex, it might be better to move it to domUtils so it is easier to test